cli/tests/integrations/test_analytics.py

```diff
@@ -115,64 +173,6 @@ def test_config_reporting_false():
         assert requests.post.call_count == 0
 
 
-@_mock
-def test_production_setting_true():
```

Comment:
This is an error in `test_marathon`, so why did you move tests in `test_analytics`?